src/style.css

```
@@ -1 +1,50 @@
-/* Styles go here */
+.header {
+  margin: 0;
+  display: flex;
+  width: 100%;
+  flex-direction: row;
+  justify-content: space-around;
```

```suggestion
  justify-content: space-between;
```
лого і навігація повинні бути рознесені по краям

---

так ніби ж в референсах вони не по краям?

---

бокові падінги це було перше, що я спробувала, але тоді в тесті пропорції чомусь дуже міняються і лінк "відео" взагалі пропадає (не в браузері)